### ISSUE TITLE
API Documentation: all links are broken

### ISSUE THREAD

**Issue by vdavyskiba** [refinery/refinerycms#3144]
1. readme.md has a link to http://www.rubydoc.info/github/refinery/refinerycms but it returns errors
2. refinerycms/wiki has a broken link to http://api.refinerycms.org/
unable to find proper link to api documantation

**Comment by parndt** [comment]
Thanks, looks like YARD is having trouble generating the docs now.

@djones are you able to fix the DNS for the api link please?

**Comment by djones** [comment]
@parndt http://api.refinerycms.org should redirect to http://www.rubydoc.info/github/refinery/refinerycms (which just happens to be broken right now). Is that correct?

**Comment by parndt** [comment]
Correct

**Comment by djones** [comment]
Done

**Comment by parndt** [comment]
@djones thanks a lot!! :smile: :heart:

**Comment by bricesanchez** [comment]
How could we fix the generating of the docs?

**Comment by bricesanchez** [comment]
🆙

**Comment by parndt** [comment]
I guess by understanding the doc generator's error :) I.e. Running it at command line and tracking down the faulty reference.

**Comment by bricesanchez** [comment]
I've no errors in local, only warnings : 

```shell
➜  refinerycms git:(master) ✗ yard
[warn]: in YARD::Handlers::Ruby::MethodHandler: Undocumentable method defined on object instance
[warn]: 	in file 'core/lib/refinery/core/configuration.rb':36:

	36: def config.register_javascript(name)

[warn]: in YARD::Handlers::Ruby::MethodHandler: Undocumentable method defined on object instance
[warn]: 	in file 'core/lib/refinery/core/configuration.rb':40:

	40: def config.register_stylesheet(*args)

[warn]: in YARD::Handlers::Ruby::MethodHandler: Undocumentable method defined on object instance
[warn]: 	in file 'core/lib/refinery/core/configuration.rb':44:

	44: def config.register_visual_editor_javascript(name)

[warn]: in YARD::Handlers::Ruby::MethodHandler: Undocumentable method defined on object instance
[warn]: 	in file 'core/lib/refinery/core/configuration.rb':48:

	48: def config.register_visual_editor_stylesheet(*args)

Files:         114
Modules:        46 (   45 undocumented)
Classes:        91 (   82 undocumented)
Constants:       5 (    5 undocumented)
Methods:       448 (  226 undocumented)
 39.32% documented
```

**Comment by bricesanchez** [comment]
Are we using the Github integration : http://rubydoc.tenderapp.com/kb/git-integration/adding-a-project-from-github ?

**Comment by parndt** [comment]
I believe so..

**Comment by bricesanchez** [comment]
Could you check it or give the rights to do it?

**Comment by parndt** [comment]
I've checked; disabled, re-enabled, set up a webhook, nothing has fixed it.
I've tried to file an issue; it said something went wrong, so I'll check in a few days.

**Comment by bricesanchez** [comment]
Fixed by #3185
